config/api.yml

```diff
@@ -1600,3 +1600,25 @@
       :delete:
       - :name: delete
         :identifier: zone_delete
+  :container_deployments:
+      :description: Container Provider Deployment
+      :identifier: container_deployment
+      :klass: ContainerDeployment
+      :methods: *70174834086080
+      :options:
+      - :collection
+      :collection_actions:
+        :post:
+        - :name: create
+          :identifier: create
+        :get:
+        - :name: collect_data
+          :identifier: container_deployment_collect_data
```

Comment:
minor nit, move the :get: section before the :post: for consistency. also move the whole :container_deployments: section sorted in the file (for my sanity). Thanks !!